Frameworks-integrations/MIGraphX/Quantizations.rst

```diff
@@ -0,0 +1,145 @@
+Running Quantized ResNet50 via MIGraphX
+========================================
+
+This example walks through the dynamo PTQ workflow for running a quantized model using torch_migraphx.
+
+Prerequisite 
++++++++++++++++
+
+You must follow the `installation instructions <https://github.com/ROCmSoftwarePlatform/torch_migraphx>`_ before using this example. 
```

Comment:
installation instructions for what?

Reply:
Installation instructions for the `torch_migraphx` library (where the link points)

Reply:
> installation instructions for what?

Updated.